pages/studentAssessmentExam/studentAssessmentExam.ejs

```diff
@@ -32,7 +32,68 @@
           </p>
           <% } %>
 
-          <% if (locals.assessment.require_honor_code) { %>
+          <% if (locals.assessment.group_work) { %>  
```

Comment:
I believe large chunks of this are copied from `studentAssessmentHomework.ejs`. Rather than a blind copy/paste, we should lift this into a partial that can be shared between those two pages.